src/OpenApi/test/Services/OpenApiDocumentServiceTests.Paths.cs

```diff
@@ -0,0 +1,158 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.OpenApi.Models;
+
+public partial class OpenApiDocumentServiceTests : OpenApiDocumentServiceTestBase
+{
+    [Fact]
+    public async Task GetOpenApiPaths_ReturnsPaths()
+    {
+        // Arrange
+        var builder = CreateBuilder();
+
+        // Act
+        builder.MapGet("/api/todos", () => { });
+        builder.MapGet("/api/users", () => { });
+
+        // Assert
+        await VerifyOpenApiDocument(builder, document =>
+        {
+            Assert.Collection(document.Paths.OrderBy(p => p.Key),
+                path =>
+                {
+                    Assert.Equal("/api/todos", path.Key);
+                    Assert.Collection(path.Value.Operations.OrderBy(o => o.Key),
```

Comment:
I think `Assert.Collection` checks the collection length - is that right?  If so, why sort a collection with only one element?  Does it make the failure message more stable or something?

Reply:
I did it to make the assertion order more stable. It doesn't matter as much for this case because there is only one HTTP method associated with the path but I wanted to keep the same assertion code for all the test cases.